tests/unit/test_stats.py

```diff
@@ -298,3 +395,18 @@ def calc_correl(x, y):
         if file_string == "time_variance_rank_correlation":
             assert np.isclose(expect_var, test_results.iloc[1].iloc[1])
         os.remove(test_file)
+
+
+def test_quantile_values_above_1():
```

Comment:
```suggestion
def test_quantile_values_not_above_1():
```

Might be clearer because we are making sure that values aren't above 1?